examples/cert-val.diag

```diff
@@ -0,0 +1,24 @@
+  {
```

Comment:
mmm, at first sight this example looks wrong.  have you checked it?

Reply:
have revisited and tidied up a bit!